.github/workflows/ci.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       - name: Install Qemu and Start Qemu Docker
         shell: bash
         run: |
+          rosinstall --catkin . ../metapackage_dependencies.rosinstall
```

Comment:
Don't do this. Use the options within the ros-industrial ci stage. Look at [this](https://github.com/uwrobotics/uwrt_mars_rover/blob/d839573db9c666b9a3f111494e068cc4947ae47f/.github/workflows/ci.yml) for reference

Reply:
ah yes, i shouldve read the comment in the rosisntall file mb